app/api/custom/orders.py

```diff
@@ -93,3 +102,73 @@ def calculate_amount():
             return UnprocessableEntityError({'source': 'discount-code'}, 'Discount Usage Exceeded').respond()
 
     return jsonify(calculate_order_amount(tickets, discount_code))
+
+
+@order_blueprint.route('/create-order', methods=['POST'])
+@jwt_required
+def create_order():
+    data = request.get_json()
+    tickets = data['tickets']
+    attendee = data['attendee']
```

Comment:
This will create single attendee. We want multiple attendees functionality

Reply:
@iamareebjamal At first, we just want multiple attendees with test info, and the information of `TicketHolder` is updated when the order is finalized. The logic for multiple attendees can be found from Line 137.

Reply:
But that's just creating attendees using `result[0]`

Reply:
@iamareebjamal That's because the dummy `TicketHolder` made during Order initialization are all created with the same info, except only their `ticket_id` which might be different for different `attendees` in the same order.

Reply:
This endpoint won't be for dummy entries. Another endpoint called `reserve-tickets` will be created for that. And mechanism of implementing it will be decided later on. This should create actual orders properly

Reply:
@iamareebjamal When the order is initialized, attendee information is not passed. It's done after the order initialization, in further steps. But as we need to reserve those tickets, we make the dummy `Ticketholders`.

Do we need to change this workflow? Like directly passing all the TicketHolder info along with the initialization of Order.

Reply:
Let me think and get back to you